api/lib/source/polkadotV0-source.js

```diff
@@ -729,20 +732,30 @@ class polkadotAPI {
         ]
       }
     ].concat(
-      proposal.seconds.map((second) => ({
-        depositer: second,
-        amount: [
-          {
-            amount: toViewDenom(this.network, proposal.balance),
-            denom: this.network.stakingDenom
+      Promise.all(
+        proposal.seconds.map(async (second) => {
+          const secondDepositerInfo = await api.derive.accounts.info(second)
+          return {
+            depositer: this.reducers.networkAccountReducer(secondDepositerInfo),
+            amount: [
+              {
+                amount: toViewDenom(this.network, proposal.balance),
+                denom: this.network.stakingDenom
+              }
+            ]
           }
-        ]
-      }))
+        })
+      )
+    )
+    const votes = await Promise.all(
+      proposal.seconds.map(async (secondAddress) => {
+        const voterInfo = await api.derive.accounts.info(secondAddress)
+        return {
+          voter: this.reducers.networkAccountReducer(voterInfo),
```

Comment:
instead of this being a unique key, why not just `identity` or `networkAccount`?

Reply:
I am not sure I understand what you mean? You mean a key like `voter` ?

Reply:
we have voter, depositer, and proposer but they are the same identity object. instead of changing the name of the key let's use the same key name. 

```
voter
depositer
proposer
```
```
networkAccount
networkAccount
networkAccount
```

Reply:
I think that would bring a lot of confusion just like that. What about `enacter`? Not sure. I think it is better using those keys, more intuitive. The other is not so clearly understandable

Reply:
I see that it makes sense standardizing, but how can you tell `account` or `identity` refers to the account voting, proposing or depositing?

Reply:
what else would it refer to?